codeexamples/autocomplete.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <script type="text/javascript" src="../lib/loop54-js-connector.js"> </script>
+    <script type="text/javascript" src="https://static.loop54.com/lib/js/loop54-js-connector.js"> </script>
```

Comment:
We don't want these to use the statically published connector. If it points to a local file, these examples can be verified that the library still works as intended when doing changes. There are probably better ways to do that than to just pointing to a relative file path, since that requires a web server running (or allowing local scripts in the browser), but it's better than not being able to do it at all.

Reply:
Cool - yeah, makes sense

Reply:
I'll have a think about this - not sure what the best way to handle this is. Perhaps just download a copy of the library and distribute it with the sample, but then you have the pain of keeping it up to date

However, maybe that's something to consider - if the library changes, perhaps the examples need to change too

Reply:
Yes, having a distributed version that you need to keep up to date would be pretty annoying. I think just referencing the built version is fine - it means you have to run `npm start` for the samples to work, but I think we can live with that.